python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -667,7 +691,7 @@ def assembleSubregion(self, coaddExposure, bbox, tempExpRefList, imageScalerList
         matching is enabled, add the background and background variance from each coaddTempExp. Remove mask
         planes listed in config.removeMaskPlanes, Finally, mean-stack
         the actual exposures using \ref afwMath.statisticsStack "statisticsStack" with outlier rejection if
-        config.doSigmaClip=True. Assign the stacked subregion back to the coadd.
+        config.statistics=MEANCLIP. Assign the stacked subregion back to the coadd.
```

Comment:
Since you have now added more statistics options, I think you need to modify this doc.  It assumes you are doing mean or clipping.